boskos/Makefile

```diff
@@ -73,10 +74,15 @@ metrics-deployment:
 service:
 	kubectl apply -f service.yaml
 
-update-config: get-cluster-credentials
+activate-prow:
+	if [[ -a "$(SERVICE_ACCOUNT)" ]] ; then\
+		gcloud auth activate-service-account --key-file="$(SERVICE_ACCOUNT)" ;\
+	fi;
+
+update-config: activate-prow get-cluster-credentials
```

Comment:
Err, actually I'm confused. Is this meant to be run by a human or by a computer? I think in general `make` should be run by humans, but I don't think humans are going to put that service account in that location, so I'm confused.

Reply:
I maybe should make another make target to make it clear? - Currently for human it will skip activate-prow thus nothing would change.

Reply:
That feels pretty magical. I'd prefer that the boskos program itself not call `make` at all.

Reply:
I'll create a script